docs/3-MODELS.md

```diff
@@ -0,0 +1,124 @@
+## 2. MODELS (Storing the collected data)
+
+The data collected from various sources, as discussed in [2-SOURCES](/docs/2-SOURCES.md), is stored in a PostgreSQL
+database. The tables are designed using Django models.
+
+All of the project's models are defined inside the app: `ports`. The models are split into three different files: `port`,
+`buildhistory` and `stats`. Each file contains the models related to the category defined by file name.
+
+#### 1. `Port` Model
+
+It stored the basic information related to a port, with the following fields:
+- `portdir`
+- `description`
+- `homepage`
+- `epoch`
+- `platforms`
+- `categories` - A many-to-many relation with `Category` model
```

Comment:
What I find confusing this in this description overall (but it might be related to our basic database design/description) is that all of the following fields:
* maintainer
* categories
* variants
are all many-to-many relationships, yet each one of them is presented differently.

Reply:
Maintainer, Dependencies and Variants have separate tables, with a foreign key relating to the `Port` model.

I guess I do not completely understand what you are trying to say here.